lib/samson/hooks.rb

```diff
@@ -42,7 +42,6 @@ class UserError < StandardError
       :build_permitted_params,
       :buildkite_release_params,
       :can,
-      :deploy_group_env,
       :deploy_group_includes,
```

Comment:
do you document the signature of each hook somewhere? would be good to say what you expect to call deploy_env with... (deploy + deploy groups)

Reply:
yeah, for now it's "look at the source" :)

Reply:
this is where a strongly typed language would help :p